src/main/java/edu/cornell/kfs/sys/dataaccess/impl/DocumentMaintenanceDaoJdbc.java

```diff
@@ -223,25 +155,25 @@ private void processQueryFinally(PreparedStatement selectStatement, ResultSet qu
             }
         }
     }

```

Comment:
This method is invoked by two distinctly different types of queries both of which could cause the SQL exceptions that are being trapped and logged. I cannot remember whether the input data elements are also included when the LOG.error is invoked.  Would there be any value in also logging the generated "selectStatementSql" when these exceptions are trapped to capture the full SQL query for troubleshooting.

Reply:
The statement/resultSet-closing code in this method was likely originally written before we had standardized on using IOUtils.closeQuietly() or try-with-resources. I don't see a need to log the on-closing failures; however, if a different error occurred prior to the closing, we would want more information about that.

I will simplify this error handling, and also add logging as appropriate to at least show what parameterized SQL was generated when an error occurs.